Source/WebCore/html/HTMLDetailsElement.cpp

```diff
@@ -130,6 +131,27 @@ bool HTMLDetailsElement::isActiveSummary(const HTMLSummaryElement& summary) cons
     return slot == m_summarySlot.get();
 }
 
+void HTMLDetailsElement::queueDetailsToggleEventTask(DetailsState oldState, DetailsState newState)
+{
+    if (m_isToggleEventTaskQueued)
+        return;
```

Comment:
I think `m_isToggleEventTaskQueued` should be entirely removed in order for event coalescing to work correctly. Otherwise, you'll always end up just emitting the first event, instead of an event containing the first old state and the last new state.